apps/Minecraft Java Lunar/uninstall

```diff
@@ -7,4 +7,8 @@ sudo rm -rf ~/lwjgl2arm32 ~/lwjgl3arm32 ~/lwjgl3arm64 ~/lwjgl2arm64 /opt/jdk/jdk
 echo "Removing Lunar Appimage"
 rm -rf ~/.local/share/applications/Lunar-Client.desktop ~/.local/bin/lunarclient.AppImage
 
+# purge old Minecraft Java dummy deb
+apt_lock_wait
+sudo apt purge -y pi-apps-c6dfb753
```

Comment:
While this solution is fine and I'm not requesting any changes, I just wanted to point out that purge_packages could technically be used here.
```
app='Minecraft Java' purge_packages || exit 1
```
This same trick can be used for `install_packages`.